src/YoutubeAtom.tsx

```diff
@@ -192,20 +210,31 @@ export const YoutubeAtom = ({
      */
     const showPlaceholder = !iframeSrc && (!hasOverlay || hasUserLaunchedPlay);
     /**
-     * Load the you tube iframe if:
-     *
-     * - We have a source string defined (i.e. We have consent)
-     *
-     * and
-     *
-     * - One of these 3 things are true
-     *      - We don't have an overlay - so we have to load the video straight away
-     *      - The user has clicked on the overlay, so load the video iframe!
-     *      - The user has moved their mouse over the overlay so lets pre load
-     *        the content
+     * Load the YouTube iframe if:
```

Comment:
I feel like this comment has become a bit redundant.

Reply:
Removed 👍